config/webpack/development.js

```diff
@@ -3,17 +3,18 @@
 process.env.NODE_ENV = process.env.NODE_ENV || 'development';
 
 const environment = require('./environment');
+
 const config = environment.toWebpackConfig();
 
 // For more information, see https://webpack.js.org/configuration/devtool/#devtool
 config.devtool = 'eval-source-map';
 
-// Inject the preact/devtools import into all the webpacker pack files (webpack entry points) that reference at least one Preact component
+// Inject the preact/debug import into all the webpacker pack files (webpack entry points) that reference at least one Preact component
 // so that Preact compoonents can be debugged with the Preact DevTools.
 config.entry = Object.entries(config.entry).reduce(
   (previous, [entryPointName, entryPointFileName]) => {
     if (/\.jsx$/.test(entryPointFileName)) {
-      previous[entryPointName] = ['preact/devtools', entryPointFileName];
+      previous[entryPointName] = ['preact/debug', entryPointFileName];
```

Comment:
`preact/debug` also hooks us up to the Preact devtools extension, so I've removed the separate `preact/devtools` here